bundler.d/assets.rb

```diff
@@ -17,4 +17,5 @@
   gem 'gridster-rails', '~> 0.1'
   gem 'jquery_pwstrength_bootstrap', '~> 1.2'
   gem 'jquery-turbolinks', '~> 2.1'
+  gem 'zeroclipboard-rails', '~> 0.1'
```

Comment:
:octocat: uses it too. Also hiding the button if flash is not installed. but basically yeah :boom: